homeassistant/components/homematic/light.py

```diff
@@ -116,5 +135,7 @@ def _init_data_struct(self):
 
         if self.supported_features & SUPPORT_COLOR:
             self._data.update({"COLOR": None})
+        if self.supported_features & SUPPORT_COLOR:
```

Comment:
That look like a copy 2 lines above?

Reply:
Fixed check with matching supported_features const